floor/config/one-pod-layout.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This is one default for a pod of 2x4 tiles.  The way the pods transmit data, they always have to have a horizontal orientation like this.